index.html

```diff
@@ -20,6 +20,14 @@ <h1 class="header_container_title"><span id="header">Real </span>Estate </h1>
           <li class="list_item_active"><a href="./index.html">Home</a></li>
           <li><a href="./about.html">About</a></li>
           <li><a href="#">Contact Us</a></li>
+
+         
+
+             <!-- The form -->
+             <form class="formSearch" action="#">
+             <input class="search" type="text" placeholder="Search.." name="search">
+             <button class="iSearch" type="submit"><i class="fa fa-search"></i></button>
```

Comment:
the button background makes the search icon look out of place. 
remove the background and border for the button, give the search icon another colour

![Screenshot from 2021-03-03 22-42-59](https://user-images.githubusercontent.com/47992412/109869677-df092800-7c71-11eb-8034-e84789fdf1ea.png)

```css
.iSearch {
  background: transparent;
  border: none;
}
```